R/eval_forecasts.R

```diff
@@ -70,7 +69,15 @@
 #'   If you want to score the median (i.e. \code{range = 0}), you still
 #'   need to include a lower and an upper estimate, so the median has to
 #'   appear twice.}
-#' @param summarised if \code{TRUE}, only one average score is returned per model
+#' @param by character vector of columns to group scoring by. The default
+#' is \code{c("model")}, but you could e.g. group over different locations
+#' or horizons. Note that a column of the corresponding name must be
+#' present in the data. If you don't want any grouping, set \code{by = NULL}
```

Comment:
i have switched to doing this with missing(argument) rather than is.null(argument) copying Seb

Reply:
The problem is if I don't set the default to be `by = "model` then this will be a breaking change. Since you were forced to have a model column before I think it's ok to force people now to switch it off explicitly if they really don't like it. But maybe that is unclear from the documentation?

Reply:
ah I hadn't realised that was the original settings. Seems fine then